lib/wsdl.js

```diff
@@ -1877,28 +1982,24 @@ WSDL.prototype._xmlnsMap = function() {
 
 var WSDL_CACHE = { };
 
-/* 
-  Have another function to load previous WSDLs as we
-  don't want this to be invoked externally (expect for tests)
-  This will attempt to fix circular dependencies with XSD files, 
-  Given
-
-    file.wsdl
```

Comment:
I like this formatting, but see now that each line should start with `*`.  Can you add that?